src/components/core/toast.scss

```
@@ -0,0 +1,39 @@
+.ctw-toast-override {
+  --toastify-color-info: #4b5563 !important;
+  --toastify-z-index: 10001 !important; /* one higher than the drawer */
+}
+
+.Toastify__close-button {
+  color: #e5e7eb !important;
+}
+
+.Toastify__toast {
+  border-radius: 8px !important;
+  padding: 16px !important;
+}
+
+.Toastify__toast-body {
+  padding: 0px 30px 0px 0px !important;
```

nit, drop units when specifying 0

```suggestion
  padding: 0 30px 0 0 !important;
```